### ISSUE TITLE
Userswitch can‘t add queue

### ISSUE THREAD

**Issue by manml** [CPqD/ofsoftswitch13#259]
Hi
I'm trying to add a queue  to userswitch.
Command to start mininet:
mn --topo linear,3 --mac --switch user --controller remote
when I trying using:
dpctl unix:/tmp/s1 queue-mod 1 4 110
it returns:
```
SENDING (xid=0xF0FF00F0):
expmodqueue{port="1", queue={q="4", props=[minrate{min rate="110"}]}}

RECEIVED (xid=0xF0FF00F0):
error{type="QUEUE_OP_FAILED", code="EPERM", dlen="56"}
```

So I changed the setting of the mininet,removed the parameter "--no-slicing"
 In mininet/mininet/node.py line 939.

` def __init__( self, name, dpopts='--no-slicing', **kwargs ):`->
` def __init__( self, name, dpopts='', **kwargs ):`

and then rebuilt mininet  with 
`make install`

and re-experiment：

`dpctl unix:/tmp/s1 queue-mod 1 4 110`

the result:

```
Feb 06 07:20:33|00001|vconn_unix|ERR|/tmp/vconn-unix.22313.0: connection to /tmp/s1 failed: No such file or directory
dpctl: Error connecting to switch unix:/tmp/s1. (No such file or directory)
```

Can anyone help ?

**Comment by nikchez01** [comment]
switch unix:/tmp/s1. (No such file or directory)
I think your ofswitch daemon has not started. Try ps -aef command to check it and check whether socket interface is corrct or not (/tmp/s1)

**Comment by ederlf** [comment]
Fixed by 7075dec54d877b497de50b78d230ffee6c14474d
